src/assets/js/scope.js

```diff
@@ -14,10 +14,12 @@ class Scope {
       staffInformation: { staff: { value: isStaff = false } = {} } = {},
       accessInformation: {
         mozilliansorg: { values: mozilliansorgGroups = {} } = {},
+        ldap: { values: ldapGroups = {} } = {},
       } = {},
     } = user || {};
     this.isStaff = isStaff;
     this.isNdaed = 'nda' in (mozilliansorgGroups || {});
+    this.isLdap = ldapGroups ? Object.keys(ldapGroups).length > 0 : false;
```

Comment:
`ldapGroups` will be `null` for non LDAP users. `Object.keys(ldapGroups)` will error there. That's non-intuitive behavior of the destructing assignment, if something is `null` the default does not apply.

But we can leverage this and just check if `ldapGroups === null` in the ternary. LDAP is the only allowed publisher of that field and if it's not null that user has LDAP.

Reply:
`this.isLdap = ldapGroups !== null;` or `this.isLdap = Boolean(ldapGroups);`  that's what it could be.

Reply:
> ldapGroups will be null for non LDAP users. Object.keys(ldapGroups) will error there.

Since `!!null === false` then `Object.keys(ldapGroups)` will never get executed, no?

But cool, wasn't sure if it could ever be an empty object, hence the ternary, so I'll remove it.